exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -58,7 +58,7 @@ func newCwLogsPusher(expConfig *Config, params component.ExporterCreateSettings)
 	}
 
 	// create CWLogs client with aws session config
-	svcStructuredLog := cwlogs.NewClient(params.Logger, awsConfig, params.BuildInfo, expConfig.LogGroupName, session)
+	svcStructuredLog := cwlogs.NewClient(params.Logger, awsConfig, params.BuildInfo, expConfig.LogGroupName, 0, session)
```

Comment:
Should this exporter also get configuration for setting a retention period?

Reply:
Hmm I think it could be helpful if the cloudwatchlogs team would like that addition